ts/WoltLabSuite/Core/Acp/Ui/DataImport/MappingReset.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Provides the program logic for the import mapping reset.
+ *
+ * @author  Tim Duesterhus
+ * @copyright  2001-2022 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Acp/Ui/DataImport/MappingReset
+ * @woltlabExcludeBundle all
+ */
+
+import * as Ajax from "../../../Ajax";
+import * as Core from "../../../Core";
+import * as UiConfirmation from "../../../Ui/Confirmation";
+
+export function setup(): void {
+  const link = document.getElementById("deleteMapping")!;
+
+  link.addEventListener("click", () => {
+    UiConfirmation.show({
+      confirm() {
+        Ajax.apiOnce({
+          data: {
+            actionName: "resetMapping",
+            className: "wcf\\system\\importer\\ImportHandler",
+          },
+          success() {
+            window.location.reload();
+          },
+          url: "index.php?ajax-invoke&t=" + Core.getXsrfToken(),
+        });
+      },
+      message: link.dataset.confirmMessage!,
+    });
+
+    return false;
```

Comment:
This has no effect and is most likely a remnant from the jQuery implementation, instead the event's `preventDefault()` should be invoked.